components/Tooltip/Tooltip.tsx

```diff
@@ -24,14 +24,30 @@ export function Tooltip({ children, ...rest }: Props) {
       {cloneElement(children, trigger)}
       {transitions(
         (style, item) =>
-          item && <TooltipContent {...tooltip} {...rest} style={style} />
+          item && (
+            <TooltipContent
+              label={<LabelWrapper>{label}</LabelWrapper>}
+              {...tooltip}
+              {...rest}
+              style={style}
+            />
+          )
       )}
     </>
   );
 }
 
 const AnimatedTooltipContent = animated(TooltipPopup);
 
+const LabelWrapper = styled.div`
```

Comment:
Added this wrapper to the tooltip label so that we can do hyphenated word breaks — we need this for example when the vote is a super long number which has no word breaks